.travis.yml

```diff
@@ -2,13 +2,15 @@
 sudo: false
 dist: xenial
 language: java
-jdk:
-  - openjdk8
-  - openjdk11
-install: true
-script:
-    # excluding wallet-template because of the unreliable JavaFX dependency
-  - gradle clean build -x :wallettemplate:compileJava
+install:
+  - sudo apt-get install -y protobuf-compiler
```

Comment:
Is this a merge error, or do you intentionally want to install the protobuf debian package again?

Reply:
Shit. It's a merge error. Sorry.

Reply:
Fixed.